examples/tabular/40_advanced/example_resampling_strategy.py

```diff
@@ -0,0 +1,151 @@
+"""
+======================
+Tabular Classification with different resampling strategy
+======================
+
+The following example shows how to fit a sample classification model
+with different resampling strategies in AutoPyTorch
+By default, AutoPyTorch uses Holdout Validation with
+a 67% train size split.
+"""
+import os
+import tempfile as tmp
+import warnings
+
+os.environ['JOBLIB_TEMP_FOLDER'] = tmp.gettempdir()
+os.environ['OMP_NUM_THREADS'] = '1'
+os.environ['OPENBLAS_NUM_THREADS'] = '1'
+os.environ['MKL_NUM_THREADS'] = '1'
+
+warnings.simplefilter(action='ignore', category=UserWarning)
+warnings.simplefilter(action='ignore', category=FutureWarning)
+
+import sklearn.datasets
+import sklearn.model_selection
+
+from autoPyTorch.api.tabular_classification import TabularClassificationTask
+from autoPyTorch.datasets.resampling_strategy import CrossValTypes, HoldoutValTypes
+
+
+############################################################################
+# Data Loading
+# ============
+def get_data():
+    X, y = sklearn.datasets.fetch_openml(data_id=40981, return_X_y=True, as_frame=True)
+    X_train, X_test, y_train, y_test = sklearn.model_selection.train_test_split(
+        X,
+        y,
+        random_state=1,
+    )
+    return X_train, X_test, y_train, y_test
+
+
+if __name__ == '__main__':
+
+    X_train, X_test, y_train, y_test = get_data()
+    ############################################################################
+    # Build and fit a classifier with default resampling strategy
+    # ==========================
```

Comment:
To make the text format work, it has to have the same number of equals as the text above it.

Reply:
okay, thanks for pointing out